curriculum/challenges/english/16-the-odin-project/top-the-box-model/the-box-model-question-d.md

```diff
@@ -0,0 +1,47 @@
+---
+id: 64a55a6102815a7d323aab8b
+title: The Box Model Question D
+challengeType: 15
+dashedName: the-box-model-question-d
+--- 
+# --description--
+
+In the Elements pane, you can see the entire HTML structure of your page. You can click on any of the elements in this pane to select that specific element. Alternatively, you can click the blue-highlighted icon shown below on the left, and hover over any element on the page.
+
+![Inspector Icon](https://cdn.statically.io/gh/TheOdinProject/curriculum/594984d7c9f9e744577f19ea475b3864e8cc7c91/html_css/v2/foundations/inspecting-html-and-css/imgs/01.png)
+
+When an element is selected, the Styles tab will show all the currently applied styles, as well as any styles that are being overwritten (indicated by a strikethrough of the text). For example, if you use the inspector to click on the “Your Career in Web Development Starts Here” header on the TOP homepage, on the right-hand side you’ll see all the styles that are currently affecting the element, as seen below:
+
+![Overwritten style](https://cdn.statically.io/gh/TheOdinProject/curriculum/f8fd38fc62578d8e8368f5303126215a492847f0/foundations/html_css/inspecting-html-and-css/imgs/03.png)
+
+
+# --question--
+
+## --assignment--
+
+Play around with Chrome Dev Tools and see if you can answer the following question.
+
+## --text--
+
+In the Style pane, what information can you view about an element when it is selected?
+
+## --answers--
+
+HTML structure and CSS rules
+
+---
+
+Styles tab
+
+---
+
+Inspector tab
+
+---
+
+Applied styles and overwritten styles.
+
+
+## --video-solution--
+
+2
```

Comment:
```suggestion
4
```

This one doesn't seem accurate either. In the Style pane, I view the styles tab?